File: src/main/java/ru/dkolmogortsev/messages/StartTask.java
package ru.dkolmogortsev.messages;

import griffon.core.event.Event;

/**
 * Created by dkolmogortsev on 2/26/17.
 */
public class StartTask extends Event
{
    public StartTask(long taskId)
    {
        super(taskId);
    }

    public long getTaskId()
    {
        return (long)getSource();
    }
}

File: src/main/kotlin/ru/dkolmogortsev/messages/StartTask.kt
package ru.dkolmogortsev.messages

import griffon.core.event.Event

/**
 * Created by dkolmogortsev on 2/26/17.
 */
class StartTask(taskId: Long) : Event(taskId) {

    val taskId: Long
        get() = getSource() as Long
}
